src/Components/Location/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { useContext } from "react";
+import { WeatherStateData } from "../../Context";
+
+const Location = () => {
+  // http://api.openweathermap.org/data/2.5/weather?q=London,uk&APPID=f1ee0c6370a1c9875b0fd546de222926
+  const context = useContext(WeatherStateData);
+  return (
+    <p className="text">
```

Comment:
Why have you used p tag instead of material UI component, remove all the html tags wherever possible